docs/class5/module11/lab2.rst

```diff
@@ -0,0 +1,73 @@
+Lab 11.2: Create Regkey Pool
+-----------------------------------
```

Comment:
make the link matching the title, this will make cloud docs rst builder job failing.
e.g.
Lab 11.2: title
-------------

Reply:
I will fix it.